### ISSUE TITLE
Reduce UC RNG sensitivity

### ISSUE THREAD

**Issue by wakystuf** [captaincobbs/ESG-Mod#1513]
Give UC a faction trait that is similar to the ecologist law (colonize anywhere), but with a -100% FIDS penalty instead
Enables them to drop sancs early so that they aren't screwed by the lack of colonizable systems nearby, but still provides no benefit to actually putting pop there without researching the relevant col tech

**Comment by Arpharnator** [comment]
makes sense, if you ignore the part where I don't think it's possible to isolate the free colonisation from ecologist.
But maybe im wrong, it's been a while since something similar has been proposed

Otherwise I think this is a good idea

**Comment by wakystuf** [comment]
I think in theory it's possible by adding that trait to the meta-prerequisite (which is already an "or"). However...it, in fact, does not seem to work, and I'm not sure why. The trait is properly applied, shows up in the GUI, etc, but then it just like...doesn't in fact enable colonization for some reason.

**Comment by wakystuf** [comment]
This now works properly! 

![Image](https://github.com/user-attachments/assets/b3e9550c-6312-4ec7-82cf-ae887e9c7f9b)
